src/main/java/Akinator.java

```diff
@@ -0,0 +1,210 @@
+        try {
+            System.out.println(textCss("span[class='back-button-text']") + " - 6");
+        } catch (Exception e){
+            System.out.print("");
+        }
+    }
+
+    public static int showTwoOptions() {
+        String yes = textCss("a[id='a_propose_yes']");
+        String no = textCss("a[id='a_propose_no']");
+        System.out.println("--------------------");
+        System.out.println("| "+yes+" - 1 | "+no+" - 2 |");
+        System.out.println("--------------------");
+
+
+        Scanner scanner = new Scanner(System.in);
+        int option = scanner.nextInt();
+
+        switch (option) {
+            case 1:
+                return 1;
+            case 2:
+                return 2;
+            default:
+                System.out.println("Этого варианта не существует. Попробуйте еще раз. Спасибо!");
+                showTwoOptions();
+        }
+        return 0;
+    }
+
+    public static void answerQuestions() throws InterruptedException, NoSuchElementException {
+        TimeUnit.SECONDS.sleep(5);
+        while (isElementPresent()) {
+            System.out.println(driver.findElement(By.xpath("//p[@class='question-text']")).getText());
+
+            showFiveOptions();
+            chooseYourOption();
+            TimeUnit.SECONDS.sleep(5);
+        }
+    }
+    //* переписав простіше
+    public static boolean isElementPresent() {
+        try {
+            driver.findElement(By.xpath("//p[@class='question-text']"));
+            return true;
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+
+    public static void chooseYourOption() {
+        Scanner scanner = new Scanner(System.in);
+        int option = scanner.nextInt();
+
+        switch (option) {
+//Потрібно зрозуміти різницю між вибором по cssSelector-y i xpath і вибрати краще.
+            case 1:
+                clickCss("a[id='a_yes']");
+                break;
+            case 2:
+                clickCss("a[id='a_no']");
+                break;
+            case 3:
+                clickCss("a[id='a_dont_know']");
+                break;
+            case 4:
+                clickCss("a[id='a_probably']");
+                break;
+            case 5:
+                clickCss("a[id='a_probaly_not']");
+                break;
+            case 6:
+                clickCss("a[id='a_cancel_answer']");
+                break;
+
+//default + break заставляє ще раз вибрати, чи все таки потрібно примусово запустити цей switch - case це зробити?
+            //* я так розумію, що в default не треба писати break. Так як break просто перериває перевірку умов.
+// Зробив так, але ще не тестував.
+            default:
+                System.out.println("Этого варианта не существует. Попробуйте еще раз. Спасибо!");
+                chooseYourOption();
+        }
+    }
+
+    public static void confirmYourHero() throws InterruptedException, NoSuchElementException {
```

Comment:
Could you minimize the usage of throws InterruptedException, NoSuchElementException, or prevent it at all?